src/pages/guidelines/color/implementation.mdx

```diff
@@ -0,0 +1,546 @@
+theme. Included in the image are the tokens that make up each layer set and are
+identified by the following key.
+
+- Base set: Magenta
+- Layer set 01: Purple
+- Layer set 02: Teal
+- Layer set 03: Cyan
+
+<Row>
+<Column colLg={8}>
+
+<Tabs>
+
+<Tab label="White">
+
+![Layer set tokens applied to dropdown in the White theme](images/color-implementation-layer-set-W.png)
+
+</Tab>
+
+<Tab label="Gray 10">
+
+![Layer set tokens applied to dropdown in the Gray 10 theme](images/color-implementation-layer-set-G10.png)
+
+</Tab>
+
+<Tab label="Gray 90">
+
+![Layer set tokens applied to dropdown in the Gray 90 theme](images/color-implementation-layer-set-G90.png)
+
+</Tab>
+
+<Tab label="Gray 100">
+
+![Layer set tokens applied to dropdown in the Gray 100 theme](images/color-implementation-layer-set-G100.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+<br />
+
+#### Example of how to apply layer set tokens in a layout
+
+Referencing the image below, the starting base level is the page area behind and
+above the tabs; it uses `$background` from the base set. The tab component is
+layered on top of the page background to create the first layer. The selected
+tab uses `$layer-01` and the unselected tabs use `$layer-accent-01` which is not
+considered a proper layer but a supporting color for `$layer` inside of
+components. The tab content area attached to the selected tab is also only one
+layer above the base and so also uses `$layer-01` as its background.
+
+In the tab's main section, the text input field is placed on top of `$layer-01`
+making it a part of the next layer level and will use tokens from the 02 layer
+set, so `$field-02` and `$border-strong-02`. Also a part of the second layer
+level are the tiles in the sub-section, this includes the tile background
+`$layer-02` as well as the border between the tiles `$border-subtle-02`.
+However, components added on top of the tiles—the text input and overflow
+menu—are considered part of third layer level and will use tokens from the 03
+layer set.
+
+Token key
+
+- Base set: Magenta
+- Layer set 01: Purple
+- Layer set 02: Teal
+- Layer set 03: Cyan
+
+<Row>
+<Column colLg={12}>
+
+<Tabs>
+
+<Tab label="White">
+
+![In context layer set token example in the White theme](images/color-implementation-layer-set-2-W.png)
+
+</Tab>
+
+<Tab label="Gray 10">
+
+![In context layer set token example in the Gray 10 theme](images/color-implementation-layer-set-2-G10.png)
+
+</Tab>
+
+<Tab label="Gray 90">
+
+![In context layer set token example in the Gray 90 theme](images/color-implementation-layer-set-2-G90.png)
+
+</Tab>
+
+<Tab label="Gray 100">
+
+![In context layer set token example in the Gray 100 theme](images/color-implementation-layer-set-2-G100.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+<br />
+
+#### Building components with layer tokens
+
+Building component with layer tokens works much like how you would have built
```

Comment:
Could the v10 references be better located under migration documentation from v10 to v11? Then this page could make the assumption that people are doing things in v11 only.

Reply:
Yeah in the next round of edits I'll try to move some of the v10 vs v11 content out of here and into migration. That's a good idea.